### ISSUE TITLE
Add a timer between multiple-character actions, like "]]" "[[" "yy"

### ISSUE THREAD

**Issue by ccaapton** [philc/vimium#2170]
In the current implementation, two consecutive ']' clicks will always trigger "next page" action, no matter how long in between these two clicks. Even there is a one-minute gap, it will always redirect to the next page. However most of time when people use these multi-character actions, they will click twice within a second, and they don't want surprises, like click "y" twice change the clipboard without any visual clue.  

It will be nice to add a timeout action to the state-machine, so these accidents will not happen. Thanks!

**Comment by smblott-github** [comment]
Agree, we should fix that.  PR welcome.  Fix would go somewhere [here](https://github.com/philc/vimium/blob/master/content_scripts/mode_key_handler.coffee).

**Comment by mrshu** [comment]
Since #2393 has been merged, can this issue be closed?

**Comment by ajwadjaved** [comment]
Hi, is this issue still open. Would love to take it up. I see that the path that Stephen mentioned isn't valid anymore @smblott-github

**Comment by philc** [comment]
While it seems like a reasonable proposal, it also seems like a rare sharp edge. Is this really worth fixing?

Also, is there a precedent of a timeout in Vim?

**Comment by gdh1995** [comment]
I like this idea, and I suggest a timeout of 30 seconds.
